app/views/podcasts/_form_main.html.erb

```diff
@@ -0,0 +1,156 @@
+<div class="col-12 mb-4">
+  <div class="form-floating">
+    <%= form.text_field :title %>
+    <%= form.label :title %>
+  </div>
+</div>
+
+<div class="col-6 mb-4">
+  <div class="form-floating">
+    <%= form.select :prx_account_uri, podcast_account_name_options(podcast.prx_account_uri) %>
+    <%= form.label :prx_account_uri %>
+  </div>
+</div>
+
+<div class="col-6 mb-4">
+  <div class="form-floating">
+    <%= form.text_field :id, disabled: true %>
+    <%= form.label :id %>
+  </div>
+</div>
+
+<div class="col-12 mb-4">
+  <div class="form-floating">
+    <%= form.text_field :subtitle %>
+    <%= form.label :subtitle %>
+  </div>
+</div>
+
+<div class="col-12 mb-4">
+  <div class="form-floating">
+    <%= form.text_area :description %>
+    <%= form.label :description %>
+  </div>
+</div>
+
+<div class="col-12 mb-4" data-controller="toggle-field">
+  <div class="form-check <%= "d-none" if podcast.summary.present? %>">
+    <%= form.check_box :add_itunes_summary, class: "form-check-input", name: nil, checked: false, data: {toggle_field_target: "check", action: "toggle-field#changeCheck"} %>
+    <%= form.label :add_itunes_summary %>
+  </div>
+
+  <div class="form-floating <%= "d-none" unless podcast.summary.present? %>">
+    <%= form.text_area :summary, data: {toggle_field_target: "field", action: "blur->toggle-field#changeField"} %>
+    <%= form.label :summary %>
+  </div>
+</div>
+
+<div class="col-6 mb-4">
+  <div class="form-floating">
+    <%= form.text_field :link %>
+    <%= form.label :link %>
+  </div>
+</div>
+
+<div class="col-6 mb-4">
+  <div class="form-floating">
+    <%= form.select :explicit, podcast_explicit_options %>
+    <%= form.label :explicit %>
+  </div>
+</div>
+
+<div class="col-6 mb-4">
+  <%# HACK: display nested validation errors %>
+  <% nested_errors = podcast.errors.full_messages_for("itunes_categories.name").join(", ") %>
+  <% klass = nested_errors.present? ? "is-invalid form-select" : "form-select" %>
+
+  <div class="form-floating" data-controller="dynamic-form">
+    <%= form.select :itunes_category, itunes_category_options, {include_blank: true}, class: klass, data: {action: "dynamic-form#change"} %>
+    <% if nested_errors.present? %><div class="invalid-feedback"><%= nested_errors %></div><% end %>
+    <%= form.label :itunes_category %>
+
+    <%# update subcategory turbo-frame when category changes %>
+    <a hidden href="<%= new_podcast_path %>" data-dynamic-form-target="link" data-turbo-frame="itunes-subcategory"></a>
```

Comment:
Here's a fun trick with turbo/stimulus...

1. You select a different category
2. The dynamic-form controller takes this new_podcast_path, adds `?itunes_category=Arts` or whatever to it, and clicks the link.
3. We GET the `podcasts#new` action, with params `{itunes_category: "Arts"}`. It re-renders this form, with an altered subcategory select.
4. Since the link had `data-turbo-frame`... we _only_ update that `"itunes-category"` frame in the HTML.  The rest of the page stays the same.